pyproject.toml

```diff
@@ -10,6 +10,7 @@ dependencies = [
     "imageio",
     "pytest",
     "pytest-benchmark",
+    "tensorstore",
```

Comment:
As a general question, do dependencies actually need to be included at the top level here, if they're installed in isolated environments with tox when running the tests anyway? I'm thinking that e.g. for the `zarr-python` tests, there's no need to install `tensorstore` in that environment.

This is probably orthogonal to this PR, and I can open an issue if you think that's best?

Reply:
Good point! They could be specified directly in the relevant tox environments, so there's no installation of un-needed dependencies. It is handy to able to `pip install .` with these dependencies for local testing though e.g. to try out different `pytest` commands to run the benchmarks. 

If you could open a separate issue - that would be great!

Reply:
Shall I open a separate issue for this @dstansby ? Or did you want to?